src/main/java/Main.java

```diff
@@ -0,0 +1,9 @@
+package main.java;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+public interface Main {
```

Comment:
The name of this interface sounds a little ambiguous to me

Reply:
If the interface is only for one specific class, then I think we would not need it unless there would be other similar classes that also hold root tasks in the future